digdag-core/src/main/java/io/digdag/core/agent/InProcessTaskServerApi.java

```diff
@@ -7,11 +7,28 @@
 import io.digdag.spi.TaskQueueClient;
 import io.digdag.core.queue.TaskQueueServerManager;
 import io.digdag.core.workflow.WorkflowExecutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Comparator;
 import java.util.List;
+import java.util.function.ToLongFunction;
+import java.util.stream.Collectors;
 
 public class InProcessTaskServerApi
     implements TaskServerApi
 {
+    private static final Logger logger = LoggerFactory.getLogger(WorkflowExecutor.class);
```

Comment:
`InProcessTaskServerApi.class` is better ?

Reply:
Oh, good catch! I moved the code from `WorkflowExecutor` carelessly.
I'll fix it soon. Thanks